services/121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.repository.ts

```diff
@@ -83,14 +67,34 @@ export class ProgramFinancialServiceProviderConfigurationRepository extends Repo
     return response;
   }
 
+  // This methods specfically does not throw as it also used to check if the property exists
+  public async getPropertyValueByName({
```

Comment:
Why not make it simpler by having getPropertiesByNames() instead? Why would one function return names and values and another very similar function only value?
Plus I would be fine with the custom repo returning entities. Then the data structure is transparent to the caller, and they can transform it to whatever they need,